solution/GraphicalDebugging/Util.cs

```diff
@@ -143,6 +143,15 @@ public static string CppRemoveCVRef(string type)
                     type = type.Remove(i - 1);
                 }
             }
+            if (type.Contains("!"))
```

Comment:
Would it make sense to use `if else` here? Is is possible to have type like `Xxx!Type{Yyy}`? Or did you use `if` just to be safe. I'm fine with that. I'm just curious.

Reply:
I actually have replicated the pattern you used in this function, that is safe indeed in cases like you mentioned. I cannot remember having seen such types though.